changelog_scratchpad

```diff
@@ -0,0 +1,386 @@
+
+- Pedia
+    - Replaced various hard-coded numbers with content script-derived calculated values
+    - Fixed placement limit documenation for InterSpecies academy
+    - Documented Planetary Starlane Drive
+    - Documented supply network propagation mechanics
+    - Additional entries for planetary focus game concept
+    - Updated or expanded articles for Spatial Distortion, Industry, Logistics, Planet Drive, Psychic Domination, Organic Hull
+    - Added an "All Species" list in the species article
+    - Improved imperial stockpile distribution documentation
+    - Added lists of native, playable, and all species
+    - Removed out-of-universe UI-related motivation for the planet drive beacon
+    - Expanded the quick start guide
+    - Elaborated on Sly description
+    - Fixed robotic shields values
+
+
+    Add stealth bonus info about colonies in SPY_PLANET_STEALTH_MOD_DESC
+    Add extra sources of detection to METER_DETECTION_VALUE_DESC
+
+
+
+- Stringtables
+    - Updated translations: French, German, Spanish
+    - Cleaned up some unused strings
+    - Avoid referring to the player in the second person ("you", "your")
+    - Added additional custom accounting labels for various effects
+    - Renamed '%SPECIES% Colony' to 'Colony of %SPECIES%'
+    - Added some missing options description strings for colours
+    - Added a link in the 'new dyson forest growing' sitrep to the Dyson Forest pedia page 
+
+- Other
+    - Many building, species, and tech balance adjustments along with new government systems and influence-constrained growth
+    - Adjusted info shared between allied and non-allied empires
+    - Reworked how allied and not-mutual-allied supply network sharing and merging handles various situations
+    - Nerfed Telepathy bonus to Psionics cost
+    - Removed the Galactic Infrastructure tech
+    - Added more and rebalanced universe creation frequency options 
+    - Fixed issues with Starlane Bore and Starlane Nexus not correctly avoiding existing starlanes when generating new ones
+    - Removed Cyborgs tech
+    - Removed redundant prerequisite of N-Dimensional Subspace tech
+    - Added the the Asteroid coating special that appears on asteroids and gives owned asteroid ships a structure boost
+    - Reworked detection levels to increment in steps of 25 instead of 25 or 50
+    - Added native species: Celestephyte, Thenian, Nightsiders, Sleepers, Khakturian
+    - Added a system special of derelict small troop ships crewed by Khakturian, similar to other derelict specials
+    - Made Laenfa sneaky, giving them a halved research cost for planet stealth
+
+
+
+#### AI
+
+- AI adopts policies
+- AI Manages influence and stability
+- AI builds Regional Administrations and Nearly Universal Translators
+- Improved invasion handling
+- AI no longer uses protection focus for military purposes
+- Many refactorings and adaptations for changed rules and content
+- Devalued Gaia special as Sly
```

Comment:
I would remove that line. I've re-written the entire planet evaluation code, which includes the fact that Gaia there are three species who cannot use Gaia (Exobots, Sly and Celest-something).